content/ruby/concepts/argf/argf.md

```diff
@@ -0,0 +1,132 @@
+---
+Title: 'ARGF and ARGV'
+Description: 'ARGF is a module in Ruby used for reading files passed as command line arguments or in standard input (STDIN) in a terminal. A Ruby script using ARGF may be thought of as implementing a subset of the functions of the Unix command cat.'
+Subjects:
+  - 'Web Development'
+Tags:
+  - 'Input'
+  - 'Files'
+CatalogContent:
+  - 'learn-ruby'
+  - 'paths/front-end-engineer-career-path'
+---
+
+ARGF is a module in Ruby used for reading files passed as command line arguments or in standard input (STDIN) in a terminal. A Ruby script using ARGF may be thought of as implementing a subset of the functions of the Unix command cat.
+
+```shell
+cat file1
+# Output: File 1 contents
+
+cat file2
+# Output: File2 contents
+
+echo "standard input" | cat
+# Output: standard input
+```
+
+The cat command can accept a filename or you can | pipe another command as standard input.
+
+## ARGF - the F is for files
+
+```ruby
+# argf.rb
+puts ARFG.read
+```
+
+The ARGF module has a method #file that you can use to recognize what file it is currently handling. Type this on the command line:
```

Comment:
```suggestion
The ARGF module has a method `.file` that can be used to recognize what file it is currently handling. Type this on the command line:
```

Is this method being used below?

Reply:
I have replaced this with a description of the `.read` method instead.